src/gobprepare/csv_stream.py

```diff
@@ -0,0 +1,97 @@
+class CSVStream():
```

Comment:
This class would be a good candidate to move in a separate GOB-Lib or something. Really generic code, and rather complex. Moving this into a separate lib repo can prevent trouble later on.